packages/venia-ui/lib/components/CategoryList/categoryList.js

```diff
@@ -48,7 +49,11 @@ const CategoryList = props => {
     if (error) {
         child = (
             <div className={classes.fetchError}>
-                Data Fetch Error: <pre>{error.message}</pre>
+                <FormattedMessage
+                    id={'categoryList.errorFetch'}
+                    defaultMessage={'Data Fetch Error:'}
```

Comment:
```suggestion
                    defaultMessage={'Data Fetch Error: '}
```

Reply:
We didn't have snaps for this before, but I think the JSX would have rendered the space between the string and the `<pre>`